configurable-rules/operation-summary-sentence-case/README.md

```diff
@@ -0,0 +1,61 @@
+# Operation summary uses sentence casing
+
+Authors:
+- [`@TaylorKrusen`](https://github.com/TaylorKrusen) Taylor Krusen (Redocly) 
+
+## What this does and why
+
+**What** -- this rule enforces a consistent patterns of capitalization on Operation summaries.  
+
+**Why** -- "Sentence casing vs title casing" may be the "tabs vs spaces" of the writing world, but being _consistent_ is often more important than being right. At Redocly, we prefer sentence casing for Operation summaries. By adding this configurable rule to our linter, we ensure our entire team writes them that way.
+
+## Code
+
+Here's how the configurable rule looks in the `redocly.yaml` file:
+
+```yaml
+rules:  
+  rule/operation-summary-sentence-case:
+    subject:
+      type: Operation
+      property: summary
+    message: "Operation summary must be sentence cased."
+    assertions: 
+      pattern: /^[A-Z]+[^A-Z]+$/
+```
+
+> [!NOTE]
+This rule can be repurposed for other fields with a single sentence, but not multiple sentences.
+
+### Rule explanation
```

Comment:
Blank line after heading please (it's a tiny thing, but some markdown tools will choke, and it's also easier to read the source with the extra vertical whitespace. We should probably have a linter for this!